src/main/java/org/sagebionetworks/web/client/security/AuthenticationControllerImpl.java

```diff
@@ -145,6 +157,16 @@ public String getCurrentUserSessionToken() {
 	}
 	
 	@Override
+	public UserBundle getCurrentUserBundle() {

```

Comment:
This method seems to only be called by tests.  If that is the case it needs to be removed.